src/main/java/com/daedafusion/security/authorization/impl/UnanimousResultAuthorizationImpl.java

```diff
@@ -26,12 +26,32 @@
 {
     private static final Logger log = Logger.getLogger(UnanimousResultAuthorizationImpl.class);
 
+    private Set<AuthorizationProvider> enabledProviders;
+
+    public UnanimousResultAuthorizationImpl()
+    {
+        addLifecycleListener(new LifecycleListener()
+        {
+            @Override
+            public void postStart()
+            {
+                Set<String> disabledProviders = Arrays.stream(
+                        Configuration.getInstance().getString("security.disabledAuthorizationProviders", "").split(","))
+                        .map(String::trim)
+                        .collect(Collectors.toSet());
+                enabledProviders = getProviders().stream()
+                        .filter(p -> !disabledProviders.contains(p.getClass().getName()))
```

Comment:
Hmmm doesn't this potentially set you up to accidentally disable providers you may not want to? Depending on what `getName()` returns

Reply:
getName returns the fully qualified classname  -- so `com.canopyiq.security.FooBar`

Reply:
Looking through docs, looks like `getName` provides the FQDN. But, you could still get in this scenario:
```
security.disabledAuthorizationProviders="com.journal.Auth0AuthProviderComplex"

...
getProviders() -> [com.journal.Auth0AuthProvider, com.journal.Auth0AuthProviderComplex]
```

In the above scenario, both of those providers would be disabled, but that might not be intended.